src/sections/Community/Handbook/faq.js

```diff
@@ -41,12 +41,35 @@ const Faq = () => {
             <h4>Q: I am not a coder, can I contribute too?</h4>
             <p>A: Some community members join to contribute to open source projects, others jump in to put those projects to use, some are here to help cultivate and steward our community, while others are here just to hangout and absorb. All members are most welcome. Be sure to introduce yourself in the Layer5<a href="https://slack.layer5.io"> slack </a>and let other community members get acquainted with you and vice-versa.</p>
 
+            <h4>Q: How should I approach assigning and working on issues within the community?</h4>
+            <p>A: To ensure efficiency and positive collaboration, please follow these guidelines:</p>
+            <ul>
+              <li>Assignment Confirmation: Verify issue validity to avoid wasted effort and address only relevant problems.</li>
+              <li>Avoiding Conflict: Treat fellow contributors kindly, respecting their contributions to foster a harmonious environment and maintain project momentum.</li>
+              <li>Minimize Maintainer Burden: Adhere to guidelines to reduce maintainers' time spent on clarifications and resolving conflicts, enabling them to focus on essential tasks.</li>
+            </ul>
+            <h4>Q: What should I consider when assigning issues to contributors?</h4>
+            <p>A: When assigning issues, follow these principles:</p>
+            <ul>
+              <li>Valid Issue: Assign issues that require attention.</li>
+              <li>Fix the Issue: Prioritize resolving them promptly.</li>
+              <li>Assignment Order: Generally chronological, with exceptions:</li>
+              <ul>
+                <li>Unqualified, unavailable, or incapable contributors.</li>
```

Comment:
@Saksham294 Let's break the first line
- Unqualified to perform this task currently.
- Unavailable and/or incapable of performing the task promptly.

Reply:
Done @thisiskaransgit